app/js/components/header/Header.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import logo from '../../../img/openmrs-with-title-small.png';
+
+export default () => (
```

Comment:
Is this restructuring a directory or adding a totally new class?

Reply:
That is not supposed to be in this PR. I have removed it.